### DISCUSSION TITLE
[Raycast]通过 Raycast 检索简悦在本地的稍后读（未发布）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3119] [服务]
# 描述

> 使用 Raycast 快速查询简悦在本地的稍后读

https://user-images.githubusercontent.com/81074/143815314-e6270ced-93e7-4e14-a708-1ce26ba756e1.mp4

# Raycast

> [Raycast](https://www.raycast.com/) 是一个 Mac 下使用的颜值超高的启动器，基本上跟 Alfred 的功能类似，但目前免费，且它的 Extensions 开发相对 Alfred 更加丰富。

# 功能

> 与 Alfred 一样提供在 Desktop 对简悦稍后读的检索功能，包括：

1. `keywords` → 检索任意内容，包括：标题，描述，备注，标签，标注的备注，标签

2. 查看当前稍后读的详细内容，如：描述，标签，备注，标注（ 标签，备注）等

3. 打开当前稍后读的原页面

4. 打开当前稍后读的本地快照

# 如何安装

> 正在提交给 Raycast，暂时无法手动安装。

# 如何使用

1. 在 Raycast 检索 `store` 后打开它的应用商店。

![image](https://user-images.githubusercontent.com/81074/143824054-ae9d7b41-703e-42d1-9002-4f1c9c812270.png)

2. 输入 `simpread` 安装即可。

![image](https://user-images.githubusercontent.com/81074/143823903-209f238b-a485-4444-aa4c-8d0668b04057.png)

# 截图

![2021-11-29_13-51-53](https://user-images.githubusercontent.com/81074/143815702-f4822899-862a-4814-8354-088237a5bba4.png)

![2021-11-29_13-35-51](https://user-images.githubusercontent.com/81074/143815341-b26cb8ba-dac9-45f9-8321-85fee9ed3aec.png)

![2021-11-29_13-39-18](https://user-images.githubusercontent.com/81074/143815342-414b37df-3a13-4974-8ab4-a34769bd1fbb.png)

![2021-11-29_13-40-45](https://user-images.githubusercontent.com/81074/143815343-ae525a66-1443-4a73-9f28-0a78d121f3c7.png)

![2021-11-29_13-41-56](https://user-images.githubusercontent.com/81074/143815347-6442438a-41c2-4c84-9dbb-d46a000d2cde.png)

# 关联

- [在 Alfred 中检索简悦的稍后读](http://ksria.com/simpread/docs/#/Alfred)

- [[uTools]通过 uTools 检索简悦在本地的稍后读](https://github.com/Kenshin/simpread/discussions/3108)

# 引申

- [当简悦变成你的知识库（离线且具有永久链接）](https://github.com/Kenshin/simpread/discussions/2221)

**Comment by DelmarHuang** [comment]
你好，这个不管我把路径填为
1. /Users/delmar/Nutstore Files/SimpRead/simpread_config.json
2. /Users/delmar/Nutstore\ Files/SimpRead/simpread_config.json
3. /Users/delmar/Nutstore Files/SimpRead
4. /Users/delmar/Nutstore\ Files/SimpRead

一直提示我让我确认是否存在simpread_config.json

<img width="426" alt="image" src="https://user-images.githubusercontent.com/54244063/169568746-177b27eb-ba9a-444a-b871-0ab74cbe4c71.png">

**Reply by Kenshin** [reply]
嗨，这个插件有问题，我也不知道原因…

开发环境是可用的，但发布后就无法使用了… 😂

**Reply by ethanchengyu** [reply]
我也是遇到这个折腾好久了，还是不行

**Comment by here-ted** [comment]
目前我把简悦稍后读用导出功能同步到了notion，然后用raycast安装notion插件实现搜索

**Reply by Kenshin** [reply]
哈哈，太能「折腾了」。 

近期就开始折腾看看为什么发布后就无法使用了… 😂
